documentation/changelog.rst

```diff
@@ -9,12 +9,14 @@ v0.10.0 | April XX, 2022
 New features
 -----------
 * Improve legibility of chart axes [see `PR #413 <http://www.github.com/FlexMeasures/flexmeasures/pull/413>`_]
+* API provides health readiness check at /api/v3_0/health/ready [see `PR #416 <http://www.github.com/FlexMeasures/flexmeasures/pull/416>`_]
 
 Bugfixes
 -----------
 
 Infrastructure / Support
 ----------------------
+* Dockerfile to run FlexMeasures API in container; also docker-compose file [see `PR #416 <http://www.github.com/FlexMeasures/flexmeasures/pull/416>`_]
```

Comment:
Not just limited to the API, right?

Reply:
Yes, it can run the CLI and UI, as well. Just not workers.